Makefile

```diff
@@ -14,8 +15,13 @@ orm.install:
 orm.status:
 	php bin/console doctrine:schema:validate --env=$(ENV)
 
-load.fixtures:
-	php bin/console doctrine:fixtures:load --no-interaction --env=$(ENV)
+### Minimal
+orm.load-minimal:
+	php bin/console doctrine:fixtures:load --group=minimal --no-interaction --env=$(ENV)
+
+orm.load-fake:
+	php bin/console doctrine:fixtures:load --group=fake --append --no-interaction --env=$(ENV)
```

Comment:
@mathieu-ducrot here it's the --append which allow not to purge the database not the purge mode in your loader